integrated_channels/moodle/models.py

```diff
@@ -29,38 +29,57 @@ class MoodleEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfiguratio
 
     moodle_base_url = models.CharField(
         max_length=255,
-        blank=True,
         verbose_name="Moodle Base URL",
         help_text=_("The base URL used for API requests to Moodle")
     )
 
-    api_token = models.CharField(
-        max_length=100,
-        null=True,
-        verbose_name="Developer Token",
+    service_short_name = models.CharField(
+        max_length=255,
+        verbose_name="Webservice Short Name",
         help_text=_(
-            "The token used to authenticate to Moodle. "
-            "Created in Moodle at Site administration/Plugins/Web services/Manage tokens"
+            "The short name for the Moodle webservice."
         )
+    )
 
+    category_id = models.IntegerField(
+        blank=True,
+        null=True,
+        verbose_name="Category ID",
+        help_text=_(
+            "The category ID for what edX courses should be associated with."
+        )
     )
 
-    wsusername = models.CharField(
```

Comment:
I named them wsusername and wspassword since those are the actual parameter names in Moodle's api.
My guess is Jeff changed them since these field names would be visible in Django admin and might confuse support or something idk.

@jlajoie why you hatin' on my field names?